eng/ci/templates/jobs/run-integration-tests.yml

```diff
@@ -0,0 +1,17 @@
+jobs:
+- job: RunIntegrationTests
+  displayName: 'Run integration tests'
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  steps:
+  - task: DotNetCoreCLI@2
+    displayName: Run End to End tests
+    inputs:
+      command: test
+      projects: |
```

Comment:
Do these tests need any secrets? If not, I recommend just lumping them in with unit tests job.